platform/ui/src/components/ViewportActionBar/ViewportActionBar.jsx

```diff
@@ -224,23 +219,36 @@ ViewportActionBar.propTypes = {
       scanner: PropTypes.string.isRequired,
     }),
   }).isRequired,
+  numCols: PropTypes.number,
 };
 
 ViewportActionBar.defaultProps = {
   showNavArrows: true,
   showPatientInfo: false,
+  numCols: 1,
 };
 
 function PatientInfo({
   patientName,
   patientSex,
   patientAge,
   MRN,
-  thickness,
+  thickness = 'N/A',
```

Comment:
Was this called out as a requirement? If this is a prop, can we set the default value in `defaults` instead of here?